vite.config.ts

```diff
@@ -1,18 +1,30 @@
+/// <reference types="vitest" />
 import path from 'path';
 import { UserConfig } from 'vite';
-import svelte from '@sveltejs/vite-plugin-svelte';
+import { svelte } from '@sveltejs/vite-plugin-svelte';
 import rewriteAll from 'vite-plugin-rewrite-all';
+import IstanbulPlugin from 'vite-plugin-istanbul';
```

Comment:
Since we'll now have a testing env, it seemed like a little stretch on to add code coverage.
We should discuss on what the requirements should be, what's important to us. Lines, 100% coverage, etc?